content/guide/dsl-import-resolver.md

```diff
@@ -23,7 +23,7 @@ class and implements:
 	* This method is called by the fetch_import method (implemented by the [AbstractImportResolver](https://github.com/cloudify-cosmo/cloudify-dsl-parser/blob/master/dsl_parser/import_resolver/abstract_import_resolver.py#L23)
 	class) during the imports parsing process, on each import url that starts with `http`, `https` or `ftp`.
 
-## Configuring the resolver in the Manager Blueprint
```

Comment:
regarding lines 11 & 16 - isn't is also used in cases other than uploading a blueprint? (cause it's called both on the client and server?)